crates/qt-build-utils/src/lib.rs

```diff
@@ -632,4 +632,33 @@ Q_IMPORT_PLUGIN({plugin_class_name});
 
         output_path
     }
+
+    /// Run [rcc](https://doc.qt.io/qt-6/resources.html) with --list on a .qrc file
+    /// and return the files found within the resource.
+    ///
+    /// This can be used to indicate to cargo when to rerun as the contents have changed
+    pub fn qrc_files(&mut self, input_file: &impl AsRef<Path>) -> Vec<PathBuf> {
```

Comment:
Does this need to be a separate method from `qrc`? How about combining them?

Reply:
Potentially, the question becomes, should `qt-build-utils` emit `cargo:rerun-if-changed` for files or is that the job of `cxx-qt-build` ? If it's fine here then it can be moved :-)

Reply:
Ah i see there are some for env vars in there already, so done this change :-)